### PR TITLE
Capstone Project

### PR DESCRIPTION
**Pull request by Enoisong**
### In this project, I:
- Add an index.html file
- Add style.css file
- Add script.js file
- Add a Loom video presenting my project

### CODE REVIEW

**Review by jerryowusu — COMMENTED** [review]
Hi @Enoisong  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that does not have any future features listed in the readme file. Kindly consider adding one or two points under the future features.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -88,8 +92,9 @@ It will be deployed after merge
 
 
 ## 🔭 Future Features <a name="future-features"></a>
-
--   
+-   Project Page
+-   Service Page
+-   Contact Page
```
Comment (mwanawabangona) [inline README.md:97]: - [ ] `Required`
- [ ] At the"Acknowledgements" section  thats where you give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).
- [ ] please make adjustments 🥂

File: README.md [hunk 2]
```diff
@@ -20,6 +20,7 @@
 - [🙏 Acknowledgements](#acknowledgements)
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
+
```
Comment (mwanawabangona) [inline README.md:23]: - [ ] `required`
- [ ] please use of GitHub Pages to deploy the web pages.
- [ ] Make sure the project is deployed properly, all pages, links, and images work as expected.

File: index.html [hunk 3]
```diff
@@ -64,10 +63,8 @@ <h1>
             This is one of the most prestigious ICT conference of the year. During the 3 days 
             of the event, you will experience the <br>integrated technological resources that ICT provides,
             through hardware, software, and telecommunications functions.
-           
           </p>
           <br />
-
           <h2>2023.03.28(MON) ~ 30(WED)</h2>
           <p>
             @ L' Meridian Resort, Abuja, Nigeria.
```
Comment (mwanawabangona) [inline index.html:70]: - [ ] `required`
- [ ] Please make sure that the spacing, colors, and font size are same.
- [ ] please remove the white background on the navbar so it should similar to the original template.
- [ ] please see screenshot below

`your template`
image

`original template`
image

File: index.html [hunk 4]
```diff
@@ -87,9 +84,8 @@ <h4>Oral Presentation</h4>
                 Be an oral presenter of the ICT conferences. 
                 present your research to a truly international audience. 
                 Presentation slots are limited, so early submission is strongly encouraged.  
-                Feel free to share your thoughts, research and experiences with experts in the field.          
-
```
Comment (mwanawabangona) [inline index.html:91]: - [ ] `required`
- [ ] please make the boxes be the same size like the original template
image

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @Enoisong ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by amon-cofie — APPROVED** [review]
Hi @Enoisong , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

🚀 Good commit messages
🚀 No linter errors
🚀 Good pull request summary
🚀 Requested changes well implemented

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**